pom.xml

```diff
@@ -27,7 +27,7 @@
   <properties>
     <revision>4.3.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.138.4</jenkins.version>
+    <jenkins.version>2.227-SNAPSHOT</jenkins.version>
```

Comment:
Isn't this too much of a jenkins version bump? Wasn't it supposed to be 2.204?

FTR I know nothing of Maven and have a really basic idea how to read POM files, so I'm probably misunderstanding something

Reply:
That was just to test the UI out, I left the commit in if anyone else wants to test it, the last commit reverts this